pkg/pillar/hypervisor/kubevirt.go

```diff
@@ -318,6 +318,11 @@ func (ctx kubevirtContext) CreateReplicaVMIConfig(domainName string, config type
 	vmi.Spec.Networks = nads
 	vmi.Spec.Domain.Devices.Interfaces = intfs
 
+	// If FML type, set the firmware bootloader to EFI
+	if config.VirtualizationMode == types.FML {
```

Comment:
@zedi-pramodh , on the API we also have `VmBootMode` https://github.com/lf-edge/eve-api/blob/main/proto/config/vm.proto#L25 which allows to select UEFI BIOS for other virtualization modes as well. However, this is not fully implemented on EVE yet. It's not necessary to implement it here at this moment (but fell free to do so), or add a `// TODO: Implement VmBootMode` comment, pls...

Reply:
Is that implemented on controller to pass in those values ?  For now I am thinking I will leave it like this and make changes as suggested by @OhmSpectator

Reply:
Yes, it's implemented in the controller (not in the UI though). As I said, it's okay to let as it is now, but please add a TODO comment so it make it easier to track when we implement VmBootMode on EVE...